README.md

```diff
@@ -269,19 +304,39 @@ See: [Computed Props](#computed-props) pattern
 
 Do not put compound conditions in `render`.
 
-```coffeescript
-#bad
-render: ->
-  if @state.happy and @state.knowsIt
-    React.DOM.div(null, "Knows what it's all about.")
-
-#good
-isLikeTotallyHappy: ->
-  @state.happy and @state.knowsIt
-
-render: ->
-  if @isLikeTotallyHappy()
-    React.DOM.div(null, "Knows what it's all about.")
+```javascript
+// bad
+render() {
+  return <div>{if (this.state.happy && this.state.knowsIt) { return "Clapping
+hands" }</div>;
+}
+
+// better
```

Comment:
Is there a good way to visually group these (`bad`, `better`, `betterer`) examples here and in other places? I know one example just goes until the next one starts, but it seems like there could be a good way to further separate them visually.